src/Mod/Fem/App/FemPostPipelinePyImp.cpp

```diff
@@ -66,18 +67,112 @@ PyObject* FemPostPipelinePy::scale(PyObject* args)
+
+            // multistep version!
+
+            if (!(PyTuple_Check(py) || PyList_Check(py))
+                || !(PyTuple_Check(list) || PyList_Check(list))) {
+
+                std::string error = std::string(
+                    "Result and value must be list of ResultObjet and number respectively.");
+                throw Base::TypeError(error);
+            }
+
+            // extract the result objects
+            Py::Sequence result_list(py);
+            Py::Sequence::size_type size = result_list.size();
+            std::vector<FemResultObject*> results;
+            results.resize(size);
+
+            for (Py::Sequence::size_type i = 0; i < size; i++) {
+                Py::Object item = result_list[i];
+                if (!PyObject_TypeCheck(*item, &(DocumentObjectPy::Type))) {
+                    std::string error =
+                        std::string("type in result list must be 'ResultObject', not ");
+                    throw Base::TypeError(error);
+                }
+                auto obj = static_cast<DocumentObjectPy*>(*item)->getDocumentObjectPtr();
+                if (!obj->isDerivedFrom<FemResultObject>()) {
+                    throw Base::TypeError("object is not a result object");
+                }
+                results[i] = static_cast<FemResultObject*>(obj);
+            }
+
+            // extract the values
+            Py::Sequence values_list(list);
+            size = values_list.size();
+            std::vector<double> values;
+            values.resize(size);
+
+            for (Py::Sequence::size_type i = 0; i < size; i++) {
+                Py::Object item = values_list[i];
+                if (!PyFloat_Check(*item)) {
+                    std::string error = std::string("Values must be float");
+                    throw Base::TypeError(error);
+                }
+                values[i] = PyFloat_AsDouble(*item);
+            }
+
+            // extract the unit
+            Base::Unit unit = *(static_cast<Base::UnitPy*>(unitobj)->getUnitPtr());
+
+            // extract the value type
+            std::string step_type = std::string(value_type);
+
+            // Finally call the c++ function!
+            getFemPostPipelinePtr()->load(results, values, unit, step_type);
+            Py_Return;
+        }
     }
 
-    App::DocumentObject* obj = static_cast<App::DocumentObjectPy*>(py)->getDocumentObjectPtr();
-    if (!obj->isDerivedFrom<FemResultObject>()) {
-        PyErr_SetString(PyExc_TypeError, "object is not a result object");
+    Py_Return;
+}
+
+
+PyObject* FemPostPipelinePy::getFilter(PyObject* args)
+{
+    if (!PyArg_ParseTuple(args, "")) {
         return nullptr;
     }
 
-    getFemPostPipelinePtr()->load(static_cast<FemResultObject*>(obj));
-    Py_Return;
+    auto filters = getFemPostPipelinePtr()->getFilter();
+    Py::List sequence;
+    for (auto filter : filters) {
+        sequence.append(Py::asObject(filter->getPyObject()));
```

Comment:
```suggestion
        sequence.append(Py::asObject(filter->getPyObject()));
```

Is this a mistake? Converting to pyobject twice?